services/web/server/tests/unit/with_dbs/02/clusters/test_models.py

```diff
@@ -0,0 +1,64 @@
+from contextlib import suppress
+from pprint import pformat
+from typing import Any, Dict, Type
+
+import pytest
+from pydantic import BaseModel, ValidationError
+from simcore_service_webserver.clusters.models import (
+    CLUSTER_ADMIN_RIGHTS,
+    CLUSTER_MANAGER_RIGHTS,
+    CLUSTER_USER_RIGHTS,
+    Cluster,
+)
+
+
+@pytest.mark.parametrize(
+    "model_cls",
+    (Cluster,),
+)
+def test_clusters_model_examples(
+    model_cls: Type[BaseModel], model_cls_examples: Dict[str, Dict[str, Any]]
+):
+    for name, example in model_cls_examples.items():
+        print(name, ":", pformat(example))
+        model_instance = model_cls(**example)
+        assert model_instance, f"Failed with {name}"
+
+
+@pytest.mark.parametrize(
+    "model_cls",
+    (Cluster,),
+)
+def test_cluster_access_rights_correctly_created_when_owner_is_not_present(
+    model_cls: Type[BaseModel], model_cls_examples: Dict[str, Dict[str, Any]]
+):
+    for example in model_cls_examples.values():
+        owner_gid = example["owner"]
+        # remove the owner from the access rights if any
+        with suppress(KeyError):
+            example.get("access_rights", {}).pop(owner_gid)
```

Comment:
why the suppress here?

Reply:
it is a test. and sometimes there is no owner_gid and that is exactly the purpose of this test.